recipes/meos/meta.yaml

```diff
@@ -0,0 +1,52 @@
+{% set name = "meos" %}
+{% set version = "1.1.0b1" %}
```

Comment:
This one falls under CFEP-05. See https://github.com/conda-forge/cfep/blob/main/cfep-05.md

Reply:
Should I just change the "b" for "rc" (in the three recipes)? I see more instructions in the spec, but only for existing feedstrocks.

Reply:
All development releases fall under CFEP-05. You can either add the `conda_build_config.yaml` and publish it in a development label, or wait/bug upstream for a stable release.

Reply:
I see. I'll talk with the rest of the team to decide whether it's worth it to publish it now or just wait a bit. Thank you!

Reply:
@ocefpaf I updated the version numbers to fix this, since the stable release of all three packages has just been published.